apps/react/src/epilog/form-adapters/contraceptivesAdapter.ts

```diff
@@ -131,18 +134,32 @@ export const formAdapter: FormAdapter<FormValues> = {
 
     return formValues;
   },
+
+  
   formValuesToEpilog: (values) => {
+    console.log("values",values);
     /* --------------------------------- Person --------------------------------- */
 
+    if (!values.person) {
```

Comment:
Do we really want the program to crash if no person or policy was provided? At the moment, this seems to me to force the user to fill out these fields before he is able to save the form. Is that expected beahvior?